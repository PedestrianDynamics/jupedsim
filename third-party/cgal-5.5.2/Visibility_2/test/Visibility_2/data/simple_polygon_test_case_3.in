# Test case 2 for simple polygons. See https://cgal.geometryfactory.com/CGAL/Members/wiki/Visibility/TestCases
8
0 0
13 0
13 4
11 2
9 4
7 2
4 2
0 5
8
0 1
1 2
2 3
3 4
4 5
5 6
6 7
7 0
2 2