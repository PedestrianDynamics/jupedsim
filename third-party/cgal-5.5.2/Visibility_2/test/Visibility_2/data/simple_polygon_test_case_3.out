# Test case 2 for simple polygons. See https://cgal.geometryfactory.com/CGAL/Members/wiki/Visibility/TestCases
7
0 0
13 0
13 2
11 2
7 2
4 2
0 5
7
0 1
1 2
2 3
3 4
4 5
5 6
6 0